addons/auth_passkey/models/auth_passkey_key.py

```diff
@@ -182,5 +182,6 @@ def make_key(self, registration=None):
             self.env.user.login, self.env.user.id,
             ip
         )
-        # Reload the web client, as its session has been invalidated, following a change of session token.
-        return {'type': 'ir.actions.client', 'tag': 'reload'}
+        new_token = self.env.user._compute_session_token(request.session.sid)
+        request.session.session_token = new_token
+        return True
```

Comment:
Shouldn't we add a landing dialog page to confirm the key was well created?

It looks weird to me that you got no confirmation the setup is completed without any confirmation.

I would just add a second form view for `auth.passkey.key.create`, next to `auth_passkey_key_create_view_form`, with a confirmiation sentence, and this `make_key` returns the action to load that second form view specifically.